invenio_subjects_lcsh/writer.py

```diff
@@ -28,6 +28,7 @@ def yaml(self, filepath):
         """
         with open(filepath, 'w') as f:
             for e in self.iterable:
+                subject = e['subject'].replace('"', r'\"')
                 f.write(f"- id: {e['id']}\n")
                 f.write(f"  scheme: {e['scheme']}\n")
-                f.write(f"  subject: {e['subject']}\n")
+                f.write(f"  subject: \"{subject}\"\n")
```

Comment:
Using this writing approach, requires us to careful about the serialization of subjects, but it still beats using the yaml library directly. The `yaml.safe_dump()` approach takes ~89secs, this approach takes ~6sec and much less memory.